### PR TITLE
Add new endpoint and refactor existing code in dbsync-api

### PR DESCRIPTION
**Pull request by Sital999**
- Add new endpoint for drepList and Proposal list
- Refactor existing code and update swagger yaml
- Update package of libcardano for node and manager

### CODE REVIEW

File: dbsync-api/swagger.yaml [hunk 1]
```diff
@@ -231,3 +231,254 @@ paths:
           description: Invalid address provided
         '500':
           description: Internal server error
+  /api/drep/list:
```
Comment (mesudip) [inline dbsync-api/swagger.yaml:234]: this can be
GET /api/drep

File: dbsync-api/swagger.yaml [hunk 2]
```diff
@@ -144,7 +144,7 @@ paths:
         '500':
           description: Internal server error.
 
-  /api/drep-details:
+  /api/drep/details:
```
Comment (mesudip) [inline dbsync-api/swagger.yaml:147]: this can be /api/drep/:id

**Review by mesudip — CHANGES_REQUESTED** [review]

File: dbsync-api/swagger.yaml (hunk 2 above)
Comment (Sital999) [reply dbsync-api/swagger.yaml]: done

File: dbsync-api/swagger.yaml (hunk 1 above)
Comment (Sital999) [reply dbsync-api/swagger.yaml]: done

**Review by mesudip — APPROVED** [review]

**Pull request merged**